docs/apps/interchain-accounts/integration.md

```diff
@@ -122,6 +122,12 @@ app.mm.SetOrderInitGenesis(
     icatypes.ModuleName,
     ...
 )
+
+// initParamsKeeper init params keeper and its subspaces
+func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino, key, tkey sdk.StoreKey) paramskeeper.Keeper {
+	...
+	paramsKeeper.Subspace(icahosttypes.SubModuleName)
```

Comment:
I see that we are also missing `paramsKeeper.Subspace(icacontrollertypes.SubModuleName)`, so we could also add it.

Reply:
@RaulBernal do you agree that we should also add this line `paramsKeeper.Subspace(icacontrollertypes.SubModuleName)` here for completeness?

Reply:
Hey @RaulBernal, you don't agree that we should also add `paramsKeeper.Subspace(icacontrollertypes.SubModuleName)` here as well?

Reply:
Oh yes sorry

Reply:
Sorry to keep repeating myself :) but could you please add the the line `paramsKeeper.Subspace(icacontrollertypes.SubModuleName)` as well if you agree?

Reply:
Sorry to keep repeating myself :) but would you mind to add the line with `paramsKeeper.Subspace(icacontrollertypes.SubModuleName)` as well if you agree?

Reply:
Done, sorry my delay 🙏